app/routes/planet_routes.py

```diff
@@ -0,0 +1,50 @@
+from flask import Blueprint, abort, make_response
+from app.models.planet import planets
+
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.get("")
+def get_all_planets():
+    result_planets = []
+    for planet in planets:
+        result_planets.append(
+            {
+                "id" : planet.id,
+                "name": planet.name,
+                "description" : planet.description,
+                "distance_from_sun" : planet.distance_from_sun
+            }
```

Comment:
Inconsistent white spaces

```suggestion
        result_planets.append(
            {
                "id": planet.id,
                "name": planet.name,
                "description": planet.description,
                "distance_from_sun": planet.distance_from_sun
            }
```